egs/wsj/s5/steps/nnet3/tdnn/train_raw.sh

```diff
@@ -0,0 +1,636 @@
+#!/bin/bash
```

Comment:
I think the raw features must be called raw waveform features or something. "raw" here is for raw nnet without transition model (HMM) and that semantic is already being used in all the nnet3 binaries with --raw option. This script also supports sparse targets (e.g. SAD, speech / silence / music / noise classification or any other classification task that does not involve a HMM). The include_log_softmax is there for the same reason.